src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Completion/CompletionListOptimizer.cs

```diff
@@ -15,7 +15,6 @@ public static VSInternalCompletionList Optimize(VSInternalCompletionList complet
             if (completionCapability is not null)
             {
                 completionList = OptimizeCommitCharacters(completionList, completionCapability);
-                completionList = OptimizeData(completionList, completionCapability);
```

Comment:
We now "pre-optimize" how we set data on completion lists. Previously this tech would look through each completion item and see if it could lift Data from completion items onto the list. This now pre-emptively does that check and only ever sets Data once in the right location so we don't need the "after-the-fact" Data optimzation step.